### PR TITLE
fix: editing a Service, the upstream info will be lost.

### LINKED ISSUE #1299: after editing a Service, the upstream info will be lost.
# Bug report

## Describe the bug

image

## How to Reproduce

1. Go to Service List;
2. Find a Service and click on Edit;
3. Update Upstream configuration;
4. Submit;
5. Edit again but without upstream configuration.

## System information

- Version: master branch

### PR DESCRIPTION
**Pull request by guoqqqi**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] Bugfix
- [ ] New feature provided
- [ ] Improve performance
- [ ] Backport patches

- Related issues
resolve #1299 
___
### Bugfix
- How to fix?
Change condition upstreamFormData.upstream_id === ‘ ’ to !upstreamFormData.upstream_id

### CODE REVIEW

File: web/src/pages/Service/Create.tsx [hunk 1]
```diff
@@ -81,13 +81,12 @@ const Page: React.FC = (props) => {
...
         notification.success({
-          message: `${
-            serviceId
+          message: `${serviceId
               ? formatMessage({ id: 'component.global.edit' })
               : formatMessage({ id: 'component.global.create' })
-          } ${formatMessage({ id: 'menu.service' })} ${formatMessage({
-            id: 'component.status.success',
-          })}`,
+            } ${formatMessage({ id: 'menu.service' })} ${formatMessage({
```
Comment (juzhiyuan) [inline web/src/pages/Service/Create.tsx:87]: format

File: web/src/pages/Service/Create.tsx [hunk 2]
```diff
@@ -71,8 +71,8 @@ const Page: React.FC = (props) => {
     };
 
     const upstreamFormData = upstreamRef.current?.getData();
-    if (upstreamFormData.upstream_id === '') {
-      data.upstream = omit(upstreamFormData, ['upstream_id']);
+    if (!upstreamFormData.upstream_id) {
```
Comment (juzhiyuan) [inline web/src/pages/Service/Create.tsx:74]: Is it possible to have something like:

```js
{
  "upstream_id": "",
  "xxx": ""
}
``` 

in `upstreamFormData`?

File: web/src/pages/Service/Create.tsx (hunk 1 above)
Comment (guoqqqi) [reply web/src/pages/Service/Create.tsx:87]: Already format

File: web/src/pages/Service/Create.tsx (hunk 2 above)
Comment (guoqqqi) [reply web/src/pages/Service/Create.tsx:74]: No, you can only choose an existing upstream or configure your own upstream without upstream_id.
image

**Review by juzhiyuan — APPROVED** [review]

**Review by LiteSun — APPROVED** [review]

**Pull request merged**